health/src/proto/protobuf/health.rs

```diff
@@ -443,3 +443,5 @@ pub fn file_descriptor_proto() -> &'static ::protobuf::descriptor::FileDescripto
         })
     }
 }
+
+pub use super::health_grpc::*;
```

Comment:
I'm curious about the small structure change. Is this line added by hand?

Reply:
No, it's added by https://github.com/tikv/grpc-rs/pull/595/files#diff-9c5e61698d6d8a8ef4a3da7075c31857769134a932ac141d4eb62f68f131aa35R176-R188